packages/web/src/common/store/profile/sagas.js

```diff
@@ -158,6 +167,11 @@ export function* fetchSolanaCollectibles(user) {
     console.log('profile has no Solana NFTs')
   }
 
+  const currentUserId = yield select(getUserId)
+  if (currentUserId === user.user_id) {
+    yield put(solNFTsFetched())
```

Comment:
If we `put` this action before the cache update, won't the saga trigger and think the collectibles aren't there yet?

Reply:
good catch!!!